Description: Relax dependencies to match the packaged versions
Forwarded: not-needed
Last-Update: 2023-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,7 +38,7 @@
 optional = true
 
 [dependencies.triple_accel]
-version = "0.3.4"
+version = "0.4"
 
 [dev-dependencies.bincode]
 version = "1.3"
